runtime/README.md

```diff
@@ -604,6 +604,29 @@ Table: `SHUTDOWN` output arguments
 | chksum       | u32       | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips_status  | u32       | Indicates if the command is FIPS approved or an error
 
+### ADD\_SUBJECT\_ALT\_NAME
+
+Provides a subject alternative name otherName. The next time certifyKey is called via InvokeDpe, the 
```

Comment:
This setting will persist until next reboot right? Should make that clear in the documentation.

Reply:
Done